packages/gatsby/src/query/queue.js

```diff
@@ -39,15 +35,10 @@ const createDevelopQueue = getRunner => {
       cb(null, newTask)
     },
     // Filter out new query jobs if that query is already running.
-    // When the query finshes, it checks the waiting map and pushes
+    // When the query finishes, it checks the waiting map and pushes
     // another job to make sure all the user changes are captured.
     filter: (job, cb) => {
-      if (processing.has(job.id)) {
-        waiting.set(job.id, job)
-        cb(`already running`)
-      } else {
-        cb(null, job)
-      }
+      cb(null, job)
```

Comment:
this seems like no-op right now, and I think we can remove `filter` completely (?)

Reply:
LOL, @vladar left same comment, so I'll just resolve mine as duplicate ;)